mxpy-up.py

```diff
@@ -321,13 +298,154 @@ def run_post_install_checks():
+
+    if any_old_export_directive:
+        # We don't perform the removal automatically (a bit risky)
+        print(f"""
+###############################################################################
+It seems that the old path "~/elrondsdk" is still configured in shell profile.
+
+Please MANUALLY remove it from the shell profile (now or after the installer script ends).
+
+Your shell profile files:
+{profile_files_formatted}
+
+The entry (entries) to remove: 
+    {old_export_directive}
+###############################################################################
+Make sure you understand the above before proceeding further.
+###############################################################################
+""")
+        confirm_continuation(interactive)
+
+    if any_new_export_directive:
+        # Note: in some (rare) cases, here we'll have false positives (e.g. if the export directive is commented out).
+        print(f"""
+###############################################################################
+It seems that the path "~/multiversx-sdk" is already configured in shell profile.
+
+To confirm this fact, check the shell profile (now or after the installer script ends). 
+
+Your shell profile files:
+{profile_files_formatted}
+
+The entry to check (it should be present): 
+    {new_export_directive}.
+###############################################################################
+Make sure you understand the above before proceeding further.
+###############################################################################
+""")
+        confirm_continuation(interactive)
+        return
+
+    choice = input(f"""
+###############################################################################
+In order to use the "mxpy" command shortcut, you have the choice to either:
+ - manually extend the PATH variable to include "~/multiversx-sdk"
+ - allow this installer script to do it for you.
+
+In order to manually extend the PATH variable, add the following line to your shell profile file upon installation:
+
+    export PATH="${{HOME}}/multiversx-sdk:${{PATH}}"
+
+Your shell profile files:
+{profile_files_formatted}
+
+Upon editing the shell profile file, you may have to RESTART THE USER SESSION for the changes to take effect.
+
+Please pick an option:
+
+a) manually extend the PATH variable (recommended for advanced users)
+b) allow this installer script to extend the PATH variable (not recommended for advanced users)
+
+""")
+    if choice not in ["a", "b"]:
+        raise InstallError("Invalid choice.")
+
+    if choice == "a":
+        return
+
+    for profile_file in profile_files:
+        with open(profile_file, "a") as file:
+            file.write(f'\n{new_export_directive}\n')
+
+    print(f"""
+###############################################################################
+The following profile files have been updated:
+{profile_files_formatted}
+
+Please RESTART THE USER SESSION.
```

Comment:
By the way, why adding to the `.profile` file and not the `.bashrc` one? If we add to the latter, we don't need to restart the user session. I imagine it is because `.profile` exists on more OSes

Reply:
Good point, handling `.bashrc` as well now.

Indeed, more experienced users will know that only the shell needs to be restarted in this case. But leaving the message "restart the user session" or even "restart the computer" could act as a helpful "fallback".